packages/scripts/test/application.unit.ts

```diff
@@ -613,6 +613,51 @@ describe('Application', function () {
                 { timeout: 5_000 }
             );
         });
+
+        it('loads config', async () => {
+            const publicConfigsRoute = 'config';
+            const app = new Application({ basePath: staticBaseWebApplicationFixturePath });
+            await app.build({
+                featureName: 'static-base-web-application/base-web-application',
+                singleFeature: true,
+                configName: 'static-base-web-application/base',
+                publicConfigsRoute: '/config',
+                staticBuild: true,
+            });
+            // disposables.add(() => app.clean());
+
+            const { close, port } = await app.run({
+                publicConfigsRoute,
+            });
+            disposables.add(() => close());
+
+            const page = await loadPage(`http://localhost:${port}/main.html`);
+
+            const configurableElem = await page.waitForSelector('#configurable', { timeout: 5_000 });
+            const text = await configurableElem?.innerText();
+            expect(text).equal('a configured message');
+        });
+
+        it('overrides config', async () => {
+            const app = new Application({ basePath: staticBaseWebApplicationFixturePath });
+            await app.build({
+                featureName: 'static-base-web-application/base-web-application',
+                singleFeature: true,
+                configName: 'static-base-web-application/base',
+                staticBuild: true,
+                configLoaderModuleName: '@fixture/static-base-web-application-feature/dist/override-config-loader'
```

Comment:
can you replace it with a request to a file in a different package? which this feature doesn't use at runtime?